test/parallel/test-fs-realpath-on-substed-drive.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const spawnSync = require('child_process').spawnSync;
+
+if (!common.isWindows) {
+  common.skip('Test for Windows only');
+  return;
+}
+let result;
+
+// create a subst drive
+const driveLetters = 'ABCDEFGHIJKLMNOPQRSTUWXYZ';
+let driveLetter;
+for (var i = 0; i < driveLetters.length; ++i) {
+  driveLetter = `${driveLetters[i]}:`;
```

Comment:
I’d maybe just call this variable `drive`… `driveLetter` sounds so much like a single character ;)